### ISSUE TITLE
NameError: uninitialized constant ToARFF::SQLiteDB even after requiring 'to-arff'

### ISSUE THREAD

**Issue by linesrunny** [dhrubomoy/to-arff#1]
Even if I require the gem, I am getting error.

```
2.3.1 :001 > require 'to-arff'
 => true 
2.3.1 :002 > ToARFF::SQLiteDB.new
NameError: uninitialized constant ToARFF::SQLiteDB
    from (irb):2
    from /Users/linesrunny/.rvm/rubies/ruby-2.3.1/bin/irb:11:in `<main>'
```

**Comment by dhrubomoy** [comment]
The issue should be fixed with commit c850dc1a5be13cc9ee5c870cf357c6a16a5fcb7c
Should be fixed in next release.
Right new the workaround is `require 'to-arff/sqlitedb'`

**Comment by dhrubomoy** [comment]
The issue should be fixed with commit c850dc1
Should be fixed in next release.
Right new the workaround is require 'to-arff/sqlitedb'
